{"name": "test_hello2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_hello.TestHello object at 0x000002DE74906B00>\n\n    def test_hello2(self):\n    \n    \n>       assert 0\nE       assert 0\n\nscripts\\test_hello.py:15: AssertionError"}, "start": 1583394222425, "stop": 1583394222425, "uuid": "3a51da6c-7dd0-400b-8e06-381ecf4d8b2a", "historyId": "1bf1c3dbd7d578643bdc2762affb01b7", "testCaseId": "2d4f9395c65ae8ee4318420c36e6960c", "fullName": "scripts.test_hello.TestHello#test_hello2", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_hello"}, {"name": "subSuite", "value": "TestHello"}, {"name": "host", "value": "DESKTOP-PTRS48T"}, {"name": "thread", "value": "17332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_hello"}]}